processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (1 of 1)
--- memsim.hex	2015-02-10 12:16:53.856717066 -0500
+++ memcpu.hex	2015-02-10 12:16:53.820716060 -0500
@@ -1,6 +1,5 @@
-:04000000340100F0D7
+:0400000000000300F9
 :0400010034020100C4
-:0400020034030200C1
 :0400030034040300BE
 :0400040034050400BB
 :040005008C26000045
@@ -15,7 +14,4 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
 :00000001FF
